src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const smallImg = document.querySelectorAll('.list-item__link');
+
+const imagesLinksArray = Array.from(smallImg).map(
+  (item) => item.getAttribute('href'),
+  {},
```

Comment:
The second argument to `Array.from` is a mapping function, not an object. You should remove the empty object `{}` as it is not needed here.